crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -442,6 +443,19 @@ impl EventTimelineItem {
         }
     }
 
+    pub fn should_boost(&self) -> bool {
+        match self.content() {
+            TimelineItemContent::Message(msg) => match msg.msgtype() {
+                MessageType::Text(text) => {
+                    let graphmes = text.body.graphemes(true).collect::<Vec<&str>>();
+                    graphmes.iter().all(|g| emojis::get(g).is_some())
```

Comment:
Would it be easy to strip any leading/trailing whitespace from the body before doing this so that we can boost like EW does?

Reply:
Yup, good idea, added it in.